account-kit/react/src/components/auth/card/eoa.tsx

```diff
@@ -204,3 +201,20 @@ export const EoaPickCard = () => {
     />
   );
 };
+
+const customErrorMessages: Record<string, CustomErrorMessage> = {
+  chainIdNotFound: {
+    heading: "The connected wallet does not support this network",
+    body: "The wallet connection failed.",
+    tryAgainCTA: "Try again with a different wallet",
+  },
+};
+
+const getCustomErrorMessage = (error: Error) => {
+  if (error.message.includes("ChainId not found")) {
+    return customErrorMessages.chainIdNotFound;
+  }
+
+  // Use default error message
+  return null;
```

Comment:
I think we should have the default error message live here and maybe rename this method to `getErrorMessageCopy` or something similar